addon-test-support/@ember/test-helpers/setup-rendering-context.js

```diff
@@ -111,8 +111,15 @@ export default function setupRenderingContext(context) {
     };
     toplevelView.setOutletState(outletState);
 
-    // TODO: make this id configurable
-    run(toplevelView, 'appendTo', '#ember-testing');
+    let rootElement;
+    // When the host app uses `setApplication` (instead of `setResolver`) the owner has
+    // a `rootElement` set on it with the element id to be used
+    if (owner._emberTestHelpersMockOwner) {
+      rootElement = '#ember-testing';
+    } else {
+      rootElement = owner.rootElement;
+    }
```

Comment:
Why is this not using the extracted helper function?

Reply:
good point, I'll double check that all supported ember versions support calling `appendTo` with an actual element (as opposed to a selector string, which is being used here), but I totally agree that it makes sense to avoid double encoding this concept (and is why I removed the complete hard coding to begin with).